package.json

```diff
@@ -10,37 +10,39 @@
     "lint": "vue-cli-service lint",
     "build:lib": "vue-cli-service build --target lib --name Cloudinary src/index.js && npm run bundlewatch",
     "build:styleguide": "rm -rf docs/*; node docs-sources/generateDocsLinks.js; vue-cli-service lint --fix; INTERNAL_DEPS=1 vue-styleguidist build",
-    "storybook:build": "vue-cli-service storybook:build -c config/storybook",
-    "storybook:serve": "vue-cli-service storybook:serve -p 6006 -c config/storybook",
-    "styleguide": "node docs-sources/generateDocsLinks.js; vue-cli-service lint --fix; INTERNAL_DEPS=1 vue-styleguidist server"
+    "styleguide": "node docs-sources/generateDocsLinks.js; vue-cli-service lint --fix; INTERNAL_DEPS=1 vue-styleguidist server",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
   },
   "main": "dist/Cloudinary.umd.js",
   "unpkg": "dist/Cloudinary.umd.min.js",
   "dependencies": {
     "cloudinary-core": "^2.10.3",
-    "cloudinary-video-player": "^1.1.1",
-    "core-js": "2.6.9",
-    "current-script-polyfill": "1.0.0"
+    "current-script-polyfill": "1.0.0",
+    "core-js": "^2.6.9"
```

Comment:
I know we should have a yarn lock file, but I'd prefer to keep it without the caret (^)

Reply:
ouch. Will put it back